src/components/IngredientCard/IngredientCard.stories.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Chip from 'components/Chip/Chip';
+import IngredientCard from 'components/IngredientCard/IngredientCard';
+
+import Milk from 'images/ingredients/milk.png';
+import Themes from 'assets/json/themes.json';
+import Yogurt from 'images/ingredients/yogurt.png';
+
+storiesOf('IngredientCard', module)
+  .add('Milk', () => (
```

Comment:
Again, will we do this for every ingredient? This is not what storybook is meant to do... We're literally changing just text.

If you want to show the theme differences, I think it would make more sense for us to use One ingredient with all the different themes.

Reply:
same.